tests/lib/qunit-assert-domequal.js

```diff
@@ -113,6 +135,52 @@ function extract( selector, message ) {
 	} );
 	result.style = getElementStyles( elem[ 0 ] );
 	result.events = $._data( elem[ 0 ], "events" );
+
+	// jQuery >=3.4.0 uses a special focus/blur handler pair
+	// needed to fix various issues with checkboxes/radio buttons
+	// as well as being able to pass data in focus triggers.
+	// However, this leaves dummy focus & blur events if any of these
+	// events were ever listened to at a particular element. There's not
+	// a lot UI can do to fix this so just skip these handlers for
+	// data comparisons in tests.
+	// See https://github.com/jquery/jquery/issues/4496
+	if ( result.events && jQueryVersionSince( "3.4.0" ) ) {
```

Comment:
Do we actually need to restrict this code to jQuery >= 3.4.0? What would happen without the check?

Reply:
I thought since this excludes some things from the compared objects, in older jQueries where there are no additional listeners we could perform a full check. There's always a risk that the workaround will not account for some real differences and by restricting it to jQuery >=3.4.0 we'd see a test error in older jQuery versions in such a case.

UI is currently tested with Core >=1.8.